libs/designsystem/src/lib/components/page/page.component.html

```diff
@@ -8,8 +8,8 @@
         [style.visibility]="hideBackButton ? 'hidden' : null"
       ></ion-back-button>
     </ion-buttons>
-    <ion-title>
-      <div class="toolbar-title hide" [class.fade-in]="toolbarTitleVisible">
+    <ion-title [class.slide-and-fade-in]="toolbarTitleVisible">
```

Comment:
The `.hide` rule still exists in the scss file, but is removed here. 

Is it used somewhere else, or can it be removed entirely?

Reply:
Left since it's still in use.